### PR TITLE
Prevent error if title is not set

### PR DESCRIPTION
**Pull request by Thoulah**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | -

I was not sure an empty string or `null` would be the preferred solution, but in case `null` is preferred this can easily be adjusted.

### CODE REVIEW

File: src/WebView.php [hunk 1]
```diff
@@ -544,7 +544,7 @@ protected function renderBodyEndHtml(bool $ajaxMode): string
      */
     public function getTitle(): string
     {
-        return $this->title;
+        return isset($this->title) ? $this->title : '';
```
Comment (bizley) [inline src/WebView.php:547]:
```suggestion
        return $this->title ?? null;
```

Comment (Thoulah) [reply src/WebView.php:547]: That is not an option with uninitialized typed properties.

Comment (bizley) [reply src/WebView.php]: https://3v4l.org/8dfRM

Comment (bizley) [reply src/WebView.php]: But yes, return type should be `?string` in that case.

**Comment by samdark** [comment]
Thanks.

**Pull request merged**